src/charm.py

```diff
@@ -117,6 +123,55 @@ def _on_config_changed(self, event: ops.HookEvent) -> None:
         """
         self.change_config(event)
         self._set_workload_version()
+        if self._charm_state.enable_mjolnir:
+            self._enable_mjolnir()
+
+    def _enable_mjolnir(self) -> None:
+        """Enable mjolnir service."""
```

Comment:
Can you expand the description - I see below a (new) admin user is registered to create a token that is than passed when registering the new mjolnir user. Would be good to have the flow described in the docstring.

Reply:
Sure! :)